### ISSUE TITLE
wrong feature name for webpki?

### ISSUE THREAD

**Issue by aviramha** [rustls/hyper-rustls#274]
In Cargo.toml:
https://github.com/rustls/hyper-rustls/blob/78d214b518e9238da26958bc37c699490332283d/Cargo.toml#L40C1-L41C1
in code:
https://github.com/rustls/hyper-rustls/blob/78d214b518e9238da26958bc37c699490332283d/src/config.rs#L35C1-L36C1

This must be an issue, no?

**Comment by ctz** [comment]
Yes that seems like a typo in Cargo.toml. In practice it doesn't have any effect (because every optional dependency implies a feature of the same name).
